components/turbot_pipes/turbot_pipes.app.mjs

```diff
@@ -1,11 +1,77 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "turbot_pipes",
-  propDefinitions: {},
+  propDefinitions: {
+    handle: {
+      type: "string",
+      label: "Handle",
+      description: "New handle of the user",
+    },
+    orgHandle: {
+      type: "string",
+      label: "Organization Handle",
+      description: "Handle of the organization",
+    },
```

Comment:
You can use the [list orgs endpoint](https://redocly.github.io/redoc/?url=https://pipes.turbot.com/api/latest/docs/openapi.json&nocors#tag/Orgs/operation/org_list) to get async options for `orgHandle`. Update Organization and Delete Organization should list async options, but not Create Organization.